var/spack/repos/builtin/packages/vasp/package.py

```diff
@@ -90,7 +135,7 @@ def edit(self, spec, prefix):
         filter_file('^FFTW[ ]{0,}?=.*$', 'FFTW ?=', 'makefile.include')
         filter_file('^MPI_INC[ ]{0,}=.*$', 'MPI_INC ?=', 'makefile.include')
         filter_file('-DscaLAPACK.*$\n', '', 'makefile.include')
-        filter_file('^SCALAPACK*$', '', 'makefile.include')
+        filter_file('^SCALAPACK.*$', '', 'makefile.include')
```

Comment:
```suggestion
        filter_file('^SCALAPACK[ ]{0,}=.*$', 'SCALAPACK ?=', 'makefile.include')
```

Reply:
Sorry I my comment was a bit cryptic, I was more thinking of something down this line.
If you just repair the regexp it will break what you have done in the `%aocc` section

Reply:
Hi @nrichart , your point is clear to me now.
addressed the suggestions in the latest push, please check.